test/browser/main.js

```diff
@@ -1,2 +1,5 @@
+process.env.NODE_ENV = 'test';
```

Comment:
This makes it so `react-router` doesn't complain when we call `Router.run()` with a static location in the browser.